code-samples/serving/hello-world/helloworld-nodejs/package.json

```diff
@@ -9,6 +9,6 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "express": "^4.20.0"
+    "express": "^4.21.0"
```

Comment:
This needs to bump the READMe otherwise we will endup with a failure as in #6118